app/updatemanager.go

```diff
@@ -209,7 +210,9 @@ func UpdateControlMapStateOnMapExpireDefault(action string) string {
 	return action
 }
 
-var UpdateControlMapStateOnActionExecutedDefault = UpdateControlMapStateActionDefault
+func UpdateControlMapStateOnActionExecutedDefault(action string) string {
+	return action
+}
```

Comment:
Ahá! You were right on Friday, I misunderstood this (and your comment, too, as I thought you were talking about `on_map_expire`).